scripts/preinstall

```diff
@@ -1,30 +1,37 @@
 #!/bin/bash
 
-PLIST="$3/Library/Preferences/com.elliotjordan.install_or_defer"
-HELPER_LD="$3/private/tmp/com.elliotjordan.install_or_defer_helper.plist"
-MAIN_LD="$3/Library/LaunchDaemons/com.elliotjordan.install_or_defer.plist"
+RESOURCE_FILES=(
+    "$3/Library/LaunchDaemons/com.elliotjordan.install_or_defer.plist"
+    "$3/Library/LaunchDaemons/com.github.mpanighetti.install-or-defer.plist"
+    "$3/Library/Preferences/com.elliotjordan.install_or_defer.plist"
+    "$3/Library/Preferences/com.github.mpanighetti.install-or-defer.plist"
+    "$3/Library/Scripts/install_or_defer.sh"
+    "$3/Library/Scripts/Install or Defer.sh"
+    "$3/Library/Scripts/install_or_defer_helper.sh"
+    "$3/Library/Scripts/Install or Defer_helper.sh"
+    "$3/private/tmp/install-or-defer"
+)
```

Comment:
Because the bundle identifier is changing, it may be a good idea to also `pkgutil --forget com.elliotjordan.install_or_defer` somewhere in the preinstall.

Reply:
Good idea, just made that change.